packages/util/src/verkle.ts

```diff
@@ -215,3 +222,46 @@ export const getVerkleTreeKeyForStorageSlot = async (
 
   return concatBytes(getVerkleStem(verkleCrypto, address, treeIndex), toBytes(subIndex))
 }
+
+export function decodeVerkleLeafBasicData(encodedBasicData: Uint8Array): VerkleLeafBasicData {
```

Comment:
Just to note that we should at some point (best still before breaking releases even if not particularly about Verkle) give the Verkle docs a bit more love and e.g. complete and clean up the docs in this `Util` module (and e.g. give methods like `decodeVerkleLeafBasicData()` some code docs).

Also the Verkle [README](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/verkle#ethereumjsverkle) should giv a somewhat more complete picture on and introduction to the library.

Reply:
I can give this a look and get some changes started.

Reply:
Got a basic description and example set up at #3630.